tests/FondOfSpryker/Glue/CompanyUserCompanyTypeRolesRestApi/CompanyUserCompanyTypeRolesRestApiConfigTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace FondOfSpryker\Zed\CompanyUserCompanyTypeRolesRestApi;
+
+use Codeception\Test\Unit;
+use FondOfSpryker\Glue\CompanyUserCompanyTypeRolesRestApi\CompanyUserCompanyTypeRolesRestApiConfig;
+use org\bovigo\vfs\vfsStream;
+
+class CompanyUserCompanyTypeRolesRestApiConfigTest extends Unit
+{
+    /**
+     * @var \FondOfSpryker\Glue\CompanyUserCompanyTypeRolesRestApi\CompanyUserCompanyTypeRolesRestApiConfig
+     */
+    protected $companyUserCompanyTypeRolesRestApiConfig;
+
+    /**
+     * @return void
+     */
+    protected function _before(): void
+    {
+        parent::_before();
+
+        $this->vfsStreamDirectory = vfsStream::setup('root', null, [
```

Comment:
@gengjozsef remove vfsStream. Mock the "get" method of the config class. Here an example: https://github.com/fond-of/spryker-log/blob/spryker_upgrade/tests/FondOfSpryker/Zed/Log/LogConfigTest.php

Reply:
thanx for the advice, i've fixed the test. Unfortunately i could not  realize it as in you're example because "onlyMethods" is not an accepted method anymore